src/App.tsx

```diff
@@ -1,203 +1,192 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { createTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { Filter } from './types/Filter';
+import { TodoItem } from './components/TodoItem';
+
+export const USER_ID = 1414;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isloading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+  const [title, setTitle] = useState('');
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [userId] = useState(USER_ID);
```

Comment:
Do you need this state? You have already variable with this value 
<img width="313" alt="image" src="https://github.com/user-attachments/assets/1ce0b187-023f-4833-9fc0-f77c907961f9">
```suggestion
```

Reply:
<img width="631" alt="image" src="https://github.com/user-attachments/assets/18edafd2-c937-4458-8e46-4cd8cb93a96f">